conda/recipes/legate-dataframe/meta.yaml

```diff
@@ -98,6 +97,8 @@ outputs:
         - numpy >=1.23,<3.0.0a0
         - rmm {{ rapids_version }}
         - cudf {{ rapids_version }}
+    ignore_run_exports_from:
+      - cupynumeric  # only a build dependency to help resolver
     run_exports:
       - {{ pin_subpackage("legate-dataframe", max_pin="x.x") }}
```

Comment:
I think I may have changed this, `legate-raft` now includes `legate-dataframe` twice, one with: `legate-dataframe >=25.1.0,<25.2.0a0`.  I guess that seems correct?  (although unnecessary, and I could either ignore the run-export or omit the manual `legate-dataframe =25.01.*,>=0.0.0.dev0` there)